app/src/org/commcare/services/CommCareSessionService.java

```diff
@@ -535,7 +542,7 @@ public void beginSubmissionProcess(int totalItems) {
 
                 // The PendingIntent to launch our activity if the user selects this notification
                 //TODO: Put something here that will, I dunno, cancel submission or something? Maybe show it live?
-                PendingIntent contentIntent = PendingIntent.getActivity(CommCareSessionService.this, 0, callable, 0);
+                PendingIntent contentIntent = PendingIntent.getActivity(CommCareSessionService.this, 0, callable, PendingIntent.FLAG_IMMUTABLE);

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/636ed8f68dcb7d308025e7f6c07ce9c3f8729d63/app/src/org/commcare/services/CommCareSessionService.java#L545"></span>Field requires API level 23 (current min is 16): <code>android.app.PendingIntent#FLAG_IMMUTABLE</code></td>
    </tr>
  </tbody>
</table>